src/login/loginctl.c

```diff
@@ -497,101 +489,164 @@ static int print_session_status_info(sd_bus *bus, const char *path, bool *new_li
 
         _cleanup_(sd_bus_error_free) sd_bus_error error = SD_BUS_ERROR_NULL;
         _cleanup_(sd_bus_message_unrefp) sd_bus_message *m = NULL;
+        _cleanup_(table_unrefp) Table *table = NULL;
         SessionStatusInfo i = {};
+        TableCell *cell;
         int r;
 
         r = bus_map_all_properties(bus, "org.freedesktop.login1", path, map, BUS_MAP_BOOLEAN_AS_BOOL, &error, &m, &i);
         if (r < 0)
                 return log_error_errno(r, "Could not get properties: %s", bus_error_message(&error, r));
 
-        if (*new_line)
-                printf("\n");
+        table = table_new_vertical();
+        if (!table)
+                return log_oom();
 
-        *new_line = true;
+        r = table_add_cell_stringf(table, &cell, "%s - %s (" UID_FMT ")", i.id, i.name, i.uid);
+        if (r < 0)
+                return table_log_add_error(r);
 
-        printf("%s - ", strna(i.id));
+        (void) table_set_color(table, cell, ansi_highlight());
+        (void) table_set_align_percent(table, cell, 100);
 
-        if (i.name)
-                printf("%s (" UID_FMT ")\n", i.name, i.uid);
-        else
-                printf(UID_FMT "\n", i.uid);
```

Comment:
Previously, we conditionalize with `i.name`, but currently not. Is it intended?

Reply:
In `user_new` we immediately return `-EINVAL` if `UserRecord.user_name` is `NULL`. Also, in `ListSessions` we don't check if `user_name` is NULL too. So I think this can be dropped.